Marsiple/App/Posts/PostsView.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  PostsView.swift
+//  Marsiple
+//
+//  Created by Matej Korman on 16/07/2018.
+//  Copyright © 2018 Matej Korman. All rights reserved.
+//
+
+import SnapKit
+
+class PostsView: UIView {
```

Comment:
Again, a very clean UI class. Seeing such clean code produced with by the current approach is quite reaffirmining.